sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v2/schema.yaml

```diff
@@ -0,0 +1,152 @@
+fields:
+- description: Unique token generated for the Firefox download. Value should never
+    be NULL.
+  mode: NULLABLE
+  name: dltoken
+  type: STRING
+- description: Time visitor on site
```

Comment:
It may be good to elaborate more on this description.

Reply:
Unfortunately I don't any more details on how GA measures the value.

Reply:
Would `visit duration on the site` be easier to follow here? The description doesn't appear to be human "readable"? Maybe could also mention if this is in seconds, milliseconds, minutes, ...?

Reply:
I'll add the units (seconds) and update the text to exactly what GA describes it as.  I do not want to deviate from that description since doing so may lead to a mis-representation of the meaning.